Library/Homebrew/diagnostic.rb

```diff
@@ -679,10 +679,16 @@ def check_git_status
 
         message = nil
 
-        {
+        taps = {
           "Homebrew/brew"          => HOMEBREW_REPOSITORY,
           "Homebrew/homebrew-core" => CoreTap.instance.path,
-        }.each do |name, path|
+        }
+
+        if Pathname.new("#{HOMEBREW_LIBRARY}/Taps/homebrew/homebrew-cask").exist?
+          taps["Homebrew/homebrew-cask"] = Pathname.new("#{HOMEBREW_LIBRARY}/Taps/homebrew/homebrew-cask")
+        end
```

Comment:
```suggestion
        cask_tap = Tap.fetch "homebrew/cask"        
        taps["Homebrew/homebrew-cask"] = cask_tap.path if cask_tap.installed?
```

Do you think this should be done on the other cask taps as well?

Reply:
@Rylan12 
> the other cask taps

they means `homebrew/cask-drivers`, `homebrew/cask-fonts` and `homebrew/cask-versions`?

Reply:
Yep!

Reply:
I agree, I will add other cask taps to `check_git_status`.